deliverables/README.md

```diff
@@ -8,61 +10,122 @@ To answer questions, please just put the answer on the next line after the quest
 
 Example:  
 
-**What is your name?**
-Sir Student of Ruby
+**What is your name?**  
+Jack Policar
 
-**What is your quest?**
-I seek the Rails  
+**What is your quest?**  To learn the basics of Ruby on Rails
+ 
 
-**What is your favorite color?**
-~~Green~~ edit:  No, blue.  Blue!  
+**What is your favorite color?**  Today, still green. 
+  
 
 For projects on Code Academy, or other external exercises, please just paste a link into your deliverables file where we can see the result.  
 
 ### Section 1: The Front End
 #### Required - Answer these questions in the deliverables file, and complete the following. 
-What is the role of HTML in a web page?  
-What is the role of CSS in a web page?  
-What is the role of Javascript in a web page?  
+**What is the role of HTML in a web page?**  <br/>--To create the skeleton of the website.   
+
+
+**What is the role of CSS in a web page?**  <br>To create the styling on the page. 
+
+
+**What is the role of Javascript in a web page?**  <br/>Java script is a and object scripting language it is used with html. It does not however incorporate or abide by any html tags or rules. 
+
+
+JavaScript, an object scripting language is used in web pages along with mark up language HTML. But JavaScript does not incorporate or abide by any HTML tags or rules.
```

Comment:
I see this is copy-pasted from http://www.xelonline.com/role_of_javaScript_in_a_website.html  

Fine to copy-paste, as long as you:  

* understand the concept  
* link to the source